es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+          street = ' Irvine Center Drive';
```

Comment:
A) Please don't modify any code outside of converting concatenations to template literals, and B) omitting the `const` keyword in front of `street` makes it an auto-global (which are _really_ bad because they're _very_ difficult to track down):

---

![image](https://user-images.githubusercontent.com/16143771/127089021-ebf0c394-30c9-4c85-a479-0c95ab980b84.png)

---

![image](https://user-images.githubusercontent.com/16143771/127089058-66ec2868-ae6d-4bfe-8b2d-f424db04b721.png)

---

![image](https://user-images.githubusercontent.com/16143771/127089082-4df84427-c7af-459f-8b63-53630f716dcd.png)